### PR TITLE
blog/trungnt2910: GSoC 2024 final blog

### PR DESCRIPTION
**Pull request by trungnt2910**
Final report, preparing for the evaluation period.

### CODE REVIEW

File: content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+That's the end of another Google Summer of Code season!
+
+I hope the Haiku community finds my contributions useful, and I look forward to deliver more
+interesting projects in the future as the OS reaches R1.
+
+# Acknowledgements
+
+First of all, I would like to thank my mentor, waddlesplash, for his help in demystifying the deep
+corners the OS and discussing solutions to more complex problems.
```
Comment (waddlesplash) [inline content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md:129]: of the OS

File: content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+type = "blog"
+title = "[GSoC 2024] Improving the Userland Debugging Experience - Final Report"
+author = "trungnt2910"
+date = "2024-08-18"
+tags = ["haiku", "software", "gsoc", "gsoc2024", "gdb", "gdbserver"]
++++
+
+# Project overview
+
+A part of Google Summer of Code 2024, this project aims to improve the userland debugging experience
```
Comment (waddlesplash) [inline content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md:11]: Shouldn't this be in past tense?

File: content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md (hunk 1 above)
Comment (trungnt2910) [reply content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md]: Fixed.

File: content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md (hunk 2 above)
Comment (trungnt2910) [reply content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md]: Whoops. Fixed that and a few following parts as well.

**Pull request merged**